sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -2531,12 +2531,6 @@ val SHUFFLE_COMPRESSION_LZ4_CHUNK_SIZE = conf("spark.rapids.shuffle.compression.
     GpuOverrides.parts.values.toSeq.sortBy(_.tag.toString).foreach(_.confHelp(asTable))
   }
   def main(args: Array[String]): Unit = {
-    if (args.length != 2) {
-      System.err.println(s"Usage: ${this.getClass.getCanonicalName}" +
-        s" {common_configs_path} {advanced_configs_path}")
-      System.exit(1)
-    }
```

Comment:
I don't think this should have been completely removed.  I can see avoiding System.exit, but why not throw an exception with a clear error message?  Removing this means we'll get something like ArrayIndexOutOfBoundsException later on which is not as helpful for debugging.

Reply:
I agree that this is good to have, but don't agree that we should do it in this PR. This is not existing code, but is what I added in this PR as I thought it's nice to add, and removed because of the compile error.

Reply:
Created https://github.com/NVIDIA/spark-rapids/issues/11333.

Reply:
We can. I just think that it would be better to do that in a separate PR given that I have already reverted this code.

Reply:
Oh right, it used to be related and then the design changed again.  Sure, I'm fine deferring this.

Reply:
Yeah, there have been some changes going back and forth. Thank you for the review and guiding me to the right direction!